### ISSUE TITLE
Image alternative text 

### ISSUE THREAD

**Issue by struthj** [SmarterApp/AP_ItemSampler#7]
Alternative Text 

- [x] Every` <img> `element must include an alt attribute that conveys the content or function of the image to screen reader users. If the image is decorative, or if the alternative text for the image is provided in nearby text, then the image should be given 'null' alternative text (alt=""). See http://webaim.org/techniques/alttext/ for details on alternative text. 
The logo at the upper left currently has alt="Smarter Balanced logo". The alt attribute should convey the full text in the image (e.g., alt="Smarter Balanced Assessment Consortium"). The word "logo" is not necessary in alternative text. Severity 3
